examples/torch/ddpg_pendulum.py

```diff
@@ -21,7 +20,15 @@
 
 
 def run_task(snapshot_config, *_):
-    """Set up environment and algorithm and run the task."""
+    """Set up environment and algorithm and run the task.
```

Comment:
can you provide an example which uses a different optimizer? That will actually allow us to compare the API before and after.

Reply:
This launcher now uses different optimizer. You can also check out the benchmark files of `vpg`, `trpo` and `ppo` to find examples of new API.